libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts

```diff
@@ -44,6 +47,7 @@ import { TigerAuthenticatedCallGuard } from "../../../types";
 import { objRefsToIdentifiers, objRefToIdentifier } from "../../../utils/api";
 import { resolveWidgetFilters } from "./widgetFilters";
 import isEmpty from "lodash/isEmpty";
+import { includes } from "lodash";
```

Comment:
Please use the partial import `import includes from "lodash/includes";` Your version should fail validation by both eslint and depcruiser.